tests/e2e/test_api.py

```diff
@@ -29,7 +29,6 @@ def test_raw_prompt():
         json={"conversation_id": conversation_id, "query": "say hello"},
         timeout=20,
     )
-    print(vars(r))
```

Comment:
this is used for debugging when the test fails, do you have a reason to remove it?

Reply:
I can debug the tests in the same way as every other test (put debug point on the line I want the execution to stop). The app just needs to be running. So this doesn't seem helpful to me.

Is there a debug scenario I don't understand?

Reply:
When i submit a PR and a test job fails this gives me the output(in the job logs) to help me understand why the test failed w/o having to go through the steps you go to rerun the test w/ a debugger or add print statements.

In an ideal world you should not have to rerun a test to understand why it failed, the test should output all relevant information as part of the failure.  And generally just the "Assert" output is not enough, you know what didn't match the expected value, but not necessarily the context needed to understand why it didn't match.